lib/index.js

```diff
@@ -25,15 +54,15 @@ set_template_directory(process.env.TEMPLATE_DIRECTORY);
  * @param {String} type - the file type
  */
 function compile_template(template_name, type) {
-  // check if the template has already been opened
-  if(!COMPILED_TEMPLATES[template_name+'.'+type]) {
-    var filepath = process.env.TEMPLATE_DIRECTORY + '/' + template_name;
-    filepath = path.resolve(filepath + '.' + type);
-    var template = fs.readFileSync(filepath, 'utf8');
-    var compiled = Handlebars.compile(template);
-    COMPILED_TEMPLATES[template_name+'.'+type] = compiled;
-  }
-  return COMPILED_TEMPLATES[template_name+'.'+type];
+    // check if the template has already been opened
+    if (!COMPILED_TEMPLATES[template_name + '.' + type]) {
+        var filepath = process.env.TEMPLATE_DIRECTORY + '/' + template_name;
```

Comment:
we don't use `4-space` for _any_ of our projects. 
Is that the `default` in your editor...? 🤔